### PR TITLE
query: fix deprecation warning for tracing

### PR DESCRIPTION
**Pull request by psarna**
Tracing code uses a deprecated mechanism for fetching the first row
when populating traces. The behavior is now fixed.

### CODE REVIEW

File: cassandra/query.py [hunk 1]
```diff
@@ -996,7 +996,8 @@ def populate(self, max_wait=2.0, wait_for_complete=True, query_cl=None):
                 SimpleStatement(self._SELECT_SESSIONS_FORMAT, consistency_level=query_cl), (self.trace_id,), time_spent, max_wait)
 
             # PYTHON-730: There is race condition that the duration mutation is written before started_at the for fast queries
-            is_complete = session_results and session_results[0].duration is not None and session_results[0].started_at is not None
+            session_row = session_results.one() if session_results else None
```
Comment (absurdfarce) [inline cassandra/query.py:999]: Very big :+1: to moving session_row declaration higher here and using it for both the wait test and subsequent operations!

File: cassandra/query.py [hunk 2]
```diff
@@ -996,7 +996,8 @@ def populate(self, max_wait=2.0, wait_for_complete=True, query_cl=None):
...
+            is_complete = session_results and session_row.duration is not None and session_row.started_at is not None
```
Comment (absurdfarce) [inline cassandra/query.py:1000]: It seems slightly odd to look at session_results again here.  Should this perhaps be:

```python
is_complete = session_row is not None and session_row.duration is not None and session_row.started_at is not None
```
What do you think?

**Comment by absurdfarce** [comment]
@psarna Thanks for the pull request!  This is a good find; I had a minor question on the PR itself but overall this looks pretty good.

Have you signed the Contributor License Agreement for contributions to DataStax open source projects?  If not you can find it at https://cla.datastax.com.  Thanks!

File: cassandra/query.py (hunk 2 above)
Comment (psarna) [reply cassandra/query.py]: Yes, absolutely, it makes more sense to check `session_row` here, thanks

**Comment by psarna** [comment]
> @psarna Thanks for the pull request! This is a good find; I had a minor question on the PR itself but overall this looks pretty good.
> 
> Have you signed the Contributor License Agreement for contributions to DataStax open source projects? If not you can find it at https://cla.datastax.com. Thanks!

Signed, thanks

**Comment by absurdfarce** [comment]
Thanks @psarna, this looks good!

**Pull request merged**